flood_adapt/config.py

```diff
@@ -0,0 +1,63 @@
+import os
+from pathlib import Path
+
+import tomli
+
+
+def set_database_root(database_root):
+    """
+    Sets the root directory for the database.
+
+    Args:
+        database_root (str): The new root directory path.
+    """
+    os.environ["DATABASE_ROOT"] = database_root
```

Comment:
Add a check verifying whether the specified path exists.
If not, `raise` an exception.